geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -88,11 +89,27 @@ object AccumuloRunner extends Runner {
 
     val params = Option(command.params)
 
+    // TODO: handle OptionalCredentialsParams
```

Comment:
Wilco - since OptionalCredentialsParams doesn't seem to be used anywhere, should I just take this out?